.github/workflows/tests.yaml

```diff
@@ -33,35 +33,46 @@ jobs:
 
     runs-on: ${{ matrix.platform }}
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Install Node
-        uses: actions/setup-node@v2
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
           cache: "yarn"
 
-      - name: Yarn Install
-        run: yarn install
+      - run: yarn install
 
-      - name: Run Tests
+      - name: Run Tests (Node v${{ matrix.node-version }})
         run: yarn -s run test:coverage
         env:
-          REPORTS_DIR:      ${{ github.workspace }}/coverage
-          ETH_RPC_URI:      ${{ secrets.ETHEREUM_RPC_URI }}
-          POLYGON_RPC_URI:  ${{ secrets.POLYGON_RPC_URI }}
-          HARMONY_RPC_URI:  "https://api.fuzz.fi/"
-          TEST_PRIVKEY_A: ${{ secrets.TEST_PRIVKEY_A }}
+          REPORTS_DIR: ${{ github.workspace }}/coverage
+          RPC_BATCH_WAIT_TIME_MS: 75
+          HARMONY_RPC_URI:   "https://api.fuzz.fi/"
+          ETH_RPC_URI:       ${{ secrets.ETHEREUM_RPC_URI }}
+          POLYGON_RPC_URI:   ${{ secrets.POLYGON_RPC_URI }}
+          INFINITE_APPROVALS_PRIVKEY:          ${{ secrets.INFINITE_APPROVALS_PRIVKEY }}
+          INFINITE_APPROVALS_PRIVKEY_ADDRESS:  ${{ secrets.INFINITE_APPROVALS_PRIVKEY_ADDRESS }}
+          BRIDGE_INTERACTIONS_PRIVKEY:         ${{ secrets.BRIDGE_INTERACTIONS_PRIVKEY }}
+          BRIDGE_INTERACTIONS_PRIVKEY_ADDRESS: ${{ secrets.BRIDGE_INTERACTIONS_PRIVKEY_ADDRESS }}
 
-      - name: Coverage report
+      - name: Generate Coverage report
         run: yarn -s run coverage:report
         env:
           REPORTS_DIR: ${{ github.workspace }}/coverage
 
-      - name: Coveralls
+      - name: "Coveralls: process coverage report"
```

Comment:
Do we need quotes here?

Reply:
with the colon, yes, but will rename instead

Reply:
Changed in [46f40a8](https://github.com/synapsecns/sdk/pull/63/commits/46f40a8dbcb414b10025f8d2013c2063343c3521)